bin/node/runtime/src/lib.rs

```diff
@@ -1478,41 +1480,45 @@ impl_runtime_apis! {
 				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef7b99d880ec681799c0cf30e8886371da95ecffd7b6c0f78751baa9d281e0bfa3a6d6f646c70792f74727372790000000000000000000000000000000000000000").to_vec().into(),
 			];
 
+			let mut info = Vec::<BenchmarkInfo>::new();
 			let mut batches = Vec::<BenchmarkBatch>::new();
 			let params = (&config, &whitelist);
```

Comment:
IMHO you should put this info into `params` and `params` should also be converted to a struct.

Reply:
yeah i agree that makes sense

Reply:
ping @ferrell-code

Reply:
ah yes I forgot :+1:, I am happy to complete assuming you want the feature added

Reply:
Yeah, please do it